README.md

```diff
@@ -610,12 +615,12 @@ As users are not created yet, it's not possible to set an owner for these secret
 
 ## Different file formats
 
-At the moment we support the following file formats: YAML, JSON, binary
+At the moment we support the following file formats: YAML, JSON, and binary.
 
 NOTE: At the moment we do not support nested data structures that
-sops support. This might change in the future:
+sops support. This might change in the future.
 
-We support the following yaml:
+We support the following YAML:
```

Comment:
I have no idea what this means. Isn't nested YAML already supported by subdirectories? Or am I missing something?

Reply:
I forgot one passage about nested keys when I implemented the subdirectory feature and it still mentions that that's not supported. Maybe you mean this?

Reply:
Should I remove this part, then?

Reply:
Yes. This should be dropped.